conf/default-vars.yaml

```diff
@@ -33,3 +33,13 @@ vars:
   heading-5-mobile-size: 14px
 
   stretch-spacing: 2px
+
+  table-cell-padding: .75rem
+  table-sm-cell-padding: .3rem
+  table-bg: transparent
+  table-bg-accent: '#f9f9f9'
```

Comment:
unless we want to start moving colors to be in context with the vars.

@dawnerd thoughts?